ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -772,6 +776,69 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'TOP Latent Defect Tower',
+      type: 'StartsUsing',
+      netRegex: { id: '7B6F', source: 'Omega', capture: false },
+      infoText: (data, _matches, output) => {
+        const myColor = data.bugRot[data.me];
+        if (data.defamationColor === myColor)
+          return output.colorTowerDefamation!({ color: myColor });
+        else if (myColor)
+          return output.colorTower!({ color: myColor });
+      },
+      run: (data) => {
+        data.bugRot = {};
+        data.latentDefectCount = data.latentDefectCount + 1;
+      },
+      outputStrings: {
+        colorTower: {
+          en: '${color} Tower',
+        },
+        colorTowerDefamation: {
+          en: '${color} Tower /w Defamation',
+        },
+        red: {
+          en: 'Red',
+        },
+        blue: {
+          en: 'Blue',
+        },
+      },
+    },
+    {
+      id: 'TOP Pass Rot',
+      type: 'GainsEffect',
+      // D65 Critical Performance Bug (blue)
+      // DC6 Critical Underflow Bug (red)
+      // Debuffs last 27s
+      netRegex: { effectId: ['D65', 'DC6'] },
+      condition: (data, matches) => {
+        data.bugRot[matches.target] = matches.effectId === 'D65' ? 'blue' : 'red';
+        return (matches.target === data.me) && data.latentDefectCount != 3;
+      },
+      delaySeconds: (data, matches) => parseFloat(matches.duration) - (data.latentDefectCount === 0 ? 4 : 7),
```

Comment:
In general, I've heard most "pass rot" calls happen right after latent defect (+defamation / stack explosions).  Do you think it's worth triggering on those abilities instead of trying to find a correct delay?

Also, fwiw, most streams I've seen have the people without rot coming to the people in the towers (who are planting).  Maybe it'd be worth just making this unconditional and telling this to everybody?

Reply:
Ah, if that's better timing I think this trigger should be turned into a collect for the two bug debuffs and then a trigger on Latent Defect Ability for the pass and maybe a 'Grab Rot' for red/green tether or and a 'Break Tether + Grab Rot' for blue tether (with exception for last blue to just break).  A callout for the red/green for break I think can be done by tracking the loss of the blue tether debuff + delay according to the vuln.

Reply:
Ok I have changed it to the ability.